internal/server/stat/golden/get_stat_all/result.json

```diff
@@ -399,49 +399,6 @@
               "provenanceDomain": "worldbank.org",
               "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/"
             },
-            {
-              "val": {
-                "1990": 249622814,
-                "1991": 252980941,
-                "1992": 256514224,
-                "1993": 259918588,
-                "1994": 263125821,
-                "1995": 266278393,
-                "1996": 269394284,
-                "1997": 272646925,
-                "1998": 275854104,
-                "1999": 279040168,
-                "2000": 282162411,
-                "2001": 284968955,
-                "2002": 287625193,
-                "2003": 290107933,
-                "2004": 292805298,
-                "2005": 295516599,
-                "2006": 298379912,
-                "2007": 301231207,
-                "2008": 304093966,
-                "2009": 306771529,
-                "2010": 309321666,
-                "2011": 311556874,
-                "2012": 313830990,
-                "2013": 315993715,
-                "2014": 318301008,
-                "2015": 320635163,
-                "2016": 322941311,
-                "2017": 324985539,
-                "2018": 326687501,
-                "2019": 328239523,
-                "2020": 331526933,
-                "2021": 332048977,
-                "2022": 333271411,
-                "2023": 334914895
-              },
-              "measurementMethod": "OECDRegionalStatistics",
-              "observationPeriod": "P1Y",
-              "importName": "OECDRegionalDemography",
-              "provenanceDomain": "oecd.org",
-              "provenanceUrl": "https://stats.oecd.org/Index.aspx?DataSetCode=REGION_DEMOGR#"
-            },
```

Comment:
Not sure why this is deleted. This is from import OECDRegionalDemography_Person_Age_StatVarAgg or OECDRegionalDemography_Mortality based on the provenanceUrl, but don't see any deletions for this import in the differ.

Reply:
OECDRegionalDemography was deprecated in March, but got out of infrequent only in this release henc ethe deletion.